### ISSUE TITLE
Is there a bug at Assets.getText

### ISSUE THREAD

**Issue by huyinghuan** [sdarnell/meteor-win#30]
in 0.7.0.1 and before
They can work well:

```
var path = Npm.require("path");
var assetsPath = path.join("test","a.txt") //the assetsPath  is test\a.txt
var str = Assets.getText(assetsPath);
console.log("str")
```

but now this  cannot read the a.txt on 0.7.0.1-win2
error is:

```
 Error: Unknown asset: test\a.txt
```

Is this a bug? Thank you!

**Comment by sdarnell** [comment]
Yes, this was a deliberate change/fix.

The problem is that Assets.getText/getBinary() don't really take a file system path, it's parameter is more like a URL and should be specified with a forward slash, e.g. test/a.txt

The bug I fixed was that it previously expected Windows path separators, but this encourages non-portable code. People might use Asset.getText("hello\there"); and it would work on Windows but as soon as it was deployed to meteor servers (unix based) it would fail.

However, clearly your use with path.join() would have been OK, but I would suggest most users of Assets have fixed/predefined paths so will have manually typed slashes.  In the end I decided that rather than automatically (and silently) convert windows slashes to forward slashes, I'd enforce forward slashes.

So in summary, just change to:

```
var assetsPath = "test/a.txt";
```

**Comment by huyinghuan** [comment]
Thank you . I got it.
